src/Team-Services-Bot.Api/Model/VstsAccount.cs

```diff
@@ -0,0 +1,39 @@
+﻿// ———————————————————————————————
+// <copyright file="VstsAccount.cs">
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+// <summary>
+// Represents a Team Services Account Information.
+// </summary>
+// ———————————————————————————————
+
+namespace Vsar.TSBot
+{
+    using System;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Represents information about VSTS Account
+    /// </summary>
+    [DataContract]
+    public class VstsAccount
```

Comment:
The reason is simple - I'd prefer to don't guess how they construct an URL for the account as soon that information is available through the API. Such approach would be neutral to further VSTS changes like they may decide to implement custom URL for the account.